modules/core/src-jvm-native/smithy4s/Timestamp.scala

```diff
@@ -256,7 +256,7 @@ object Timestamp extends TimestampCompanionPlatform {
 
   def showFormat(format: TimestampFormat): String = format match {
     case TimestampFormat.DATE_TIME =>
-      "date-time timestamp (YYYY-MM-DDThh:mm:ss.sssZ)"
+      "date-time timestamp (YYYY-MM-ddThh:mm:ss.sssZ)"
     case TimestampFormat.EPOCH_SECONDS => "epoch-second timestamp"
     case TimestampFormat.HTTP_DATE =>
       "http-date timestamp (EEE, dd MMM yyyy HH:mm:ss.sss z)"
```

Comment:
**thought:** I think this should also be `hh` (24-h hour) and not `HH` (12-h hour)

Reply:
agreed